src/main/scala/it/pagopa/interop/authorizationserver/api/impl/AuthApiServiceImpl.scala

```diff
@@ -76,6 +80,7 @@ final case class AuthApiServiceImpl(
       clientUUID    <- checker.subject.toFutureUUID
       keyWithClient <- getTokenGenerationBundle(clientUUID, checker.kid)
       _             <- verifyClientAssertion(keyWithClient, checker)
+      _             <- rateLimiter.rateLimiting(keyWithClient.client.consumerId)
```

Comment:
here most of the work has already been done, so this rate limiting does not actually prevent loading our systems, but unfortunately this check cannot be performed earlier

Reply:
Could be useful to work also on IP?

Reply:
that job should be already done by AWS WAF

Reply:
This will prevent DOS but not DDOS 🤔